src/neptune_tensorboard/integration/__init__.py

```diff
@@ -13,3 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+__all__ = ["enable_tensorboard_logging", "__version__"]
+
+from importlib.util import find_spec
+
+from .tensorflow_integration import patch_tensorflow
+from .version import __version__
+
+
+def enable_tensorboard_logging(run, *, base_namespace="tensorboard"):
+    if find_spec("tensorflow"):
```

Comment:
What about some warning in case of lack of `tensorflow`?

Reply:
We plan to support other frameworks which also have tensorboard support like PyTorch and Tensorboardx (which would be added seperately), so we don't want to warn if tensorflow is missing.

Reply:
How about a more generalized approach?
```python
if find_spec("tensorflow") or find_spec("tensorflow-macos"):
    ...
if find_spec("tensorboardx"):
    ...
else:
    logger.warning("None of the supported packages was installed.")
```

Reply:
Updated